news/2024/2024-06-24-4-digit-world-cup-2024-concludes.md

```diff
@@ -0,0 +1,125 @@
+
+**Shurelia** has started, despite being an osu!catch player, making tournament maps for osu! recently, including a total of three for this year's 4WC.
+
+---
+
+*What brought you into mapping? Why have you decided to create custom maps for tournaments such as 4WC 2024?*
+
+> The reason why I started mapping for osu! standard specifically was due to being a heavy convert player in catch and wanting to gain #1 on a song that I really like. But the reason why I WANT to be better in mapping is because I got captivated by [Lan Wing](https://osu.ppy.sh/users/467860)'s beatmap of [t=NODE - Four Seasons](https://osu.ppy.sh/beatmapsets/285718#osu/645242). This map moves me so much that I randomly shed tears despite having no clue about osu! standard at all due to how beautiful the map and the song is.
+> 
+> I decided to create custom maps because I just love watching people playing my maps, moreover when it's in a big tournament such as 4WC 2024. It also lets me learn many things to improve my mapping knowledge gameplay wise by learning what the players think about my map when they try it, because as I mentioned, I do not play osu! standard at all and thus it's obvious that I don't testplay my own maps at all. Custom mapping is such a good way to improve myself, at least that's what I'm thinking.
+
+*How does it feel to have your maps be played in high-stake tournaments? Does it motivate a mapper to follow the tournament more closely or only if their map is played?*
+
+> It feels great! Unfortunately, due to my circumstances, my interest with osu! standard tournaments isn't as high as people might think and thus I only watch a match if my map got picked so I can see how the chat or the players reacted to it.
+
+## Statistics
+
+A lot happened this tournament that can be shown off by statistics! We've compiled all of the per-round statistics into a [neat folder](https://drive.google.com/drive/u/0/folders/1dvBabKdU7V1nx1wEMhINi89NSdBvDU5f), but here are some extra picks that deserve their own mentions:
+
+- There were a total of 33 scores set worth more than 500pp, the highest of which being [scylla](https://osu.ppy.sh/users/9405745)'s 651pp full combo on [BABYMETAL - Arkadia \[Extreme\]](https://osu.ppy.sh/beatmapsets/1695872#osu/3644487) in Sweden's [Semifinals match against Brazil](https://osu.ppy.sh/community/matches/114120011).
+- We saw a total of 16 seed upsets out of 62 matches played, the biggest of which were:
+  - [Chile (5) vs. Spain (28)](https://osu.ppy.sh/community/matches/113860341) in the Round of 32, in which 98% of pick'em players incorrectly predicted Chile to win.
+  - [South Korea (2) vs. China (14)](https://osu.ppy.sh/community/matches/114131751) in the Semifinals, notable for being very late in the bracket to have such an upset.
```

Comment:
vod links instead of mp links?

Reply:
prolly better yeah, will update once they're all uploaded (later today)